src/jit/block.h

```diff
@@ -893,8 +892,8 @@ struct BasicBlock : private LIR::Range
     BlockSet    bbReach; // Set of all blocks that can reach this one
     BasicBlock* bbIDom;  // Represent the closest dominator to this block (called the Immediate
                          // Dominator) used to compute the dominance tree.
-    unsigned bbDfsNum;   // The index of this block in DFS reverse post order
```

Comment:
There are some references of this field that were not deleted, at least:
https://github.com/dotnet/coreclr/blob/4fe26a4653b5dcc1259b26a7439f96f452cb0d9e/src/jit/block.cpp#L1514

The build doesn't fail because it is under `MEASURE_BLOCK_SIZE`.

Reply:
Huh, what in the world is this thing? I've never seen something as nonsensical as `DisplayStaticSizes`, I think I'll just delete the whole thing.